src/terrain/clients/data_info.clj

```diff
@@ -77,38 +77,27 @@
   [^String user ^String dir]
   (cr/ensure-created user dir))
 
-(defn get-public-data-user
```

Comment:
Most of the changes are moving this function - keeping this here, ended up with a circular reference, so I moved it